Model/OrderWithErrorImporter.php

```diff
@@ -50,7 +50,7 @@ public function __construct(
      */
     public function execute() : Info
     {
-        for ($i = 1; $i <= ceil($this->orderLogRepository->getCount() / 200 ); $i += 1) {
+        for ($i = 1; $i <= ceil($this->orderLogRepository->getCount() / 200); $i += 1) {
```

Comment:
Btw, shouldnt the ceil be calculated before the loop?

Reply:
or just value returned by `\Magento\Framework\Data\Collection::getLastPageNumber()`, called before loop

Reply:
Done